### ISSUE TITLE
Hyperparameter tuning to do

### ISSUE THREAD

**Issue by olivboom** [msc-acse/acse-9-independent-research-project-OliverJBoom#21]
Ranked in priority

-   Window size
-   Network parameters 
     -  Hidden Layers Length
     -  Hidden Layer Depth
     -  LSTM Layer Depth

-  Learning Rate
-  Momentum
-  Weight Decay

**Comment by olivboom** [comment]
Left:
    -    Optimiser
    -    Learning Rate
    -    Momentum
    -    Weight Decay

And these aren't high priority

**Comment by olivboom** [comment]
Other priorities, will not tune those
